_data/simple-icons.json

```diff
@@ -5180,6 +5180,11 @@
             "source": "https://graphql.org/brand",
             "guidelines": "https://graphql.org/brand"
         },
+        {
+            "title": "GrapheneOS",
+            "hex":"000000",
+            "source":"https://grapheneos.org"
```

Comment:
@adamrusted it is listed once but the linter sees two of each `hex` and `source`

Reply:
Weird. Looks like it's not in the correct place alphabetically (currently after GraphQL) - that might be causing some of this weirdness.